src/arguments.h

```diff
@@ -45,9 +45,9 @@ struct Argument
 
 enum RefQualifierType
```

Comment:
You mentioned in the description "Modernized the usage of enum class for better type safety" should this enum not have the word class?

Reply:
Thank you for your observation. You are correct, the enum should indeed have the word class for better type safety. I've fixed this issue in the code.